pennylane_lightning/core/simulators/lightning_qubit/bindings/LQubitBindings_nb.hpp

```diff
@@ -188,6 +188,19 @@ void registerBackendSpecificStateVectorMethods(PyClass &pyclass) {
     // Register sparse matrix operators.
     registerSparseMatrixOperators<StateVectorT>(pyclass);
 
+    // Add updateData method for LQubit
```

Comment:
Is there any benefit to moving `updateData` to be defined separately in each device rather than in the main bindings file?

Reply:
Currently, LKokkos does not expose the updateData method. Additionally, the function signature differs slightly from those of other devices.